packages/api/src/hooks/useMT5AccountsList.ts

```diff
@@ -33,8 +33,14 @@ const useMT5AccountsList = () => {
             loginid: account.login,
             /** The platform of the account */
             platform: 'mt5',
+            /** The balance of the account in currency format. */
+            display_balance: `${Intl.NumberFormat(authorize_data?.preferred_language || 'en-US', {
```

Comment:
```suggestion
            display_balance: `${Intl.NumberFormat(authorize_data?.preferred_language || 'en-US', {
```
Add a handler if BE ever returns a wrong locale format: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/Locale/language